desktop-widgets/subsurfacewebservices.cpp

```diff
@@ -185,11 +185,15 @@ static DiveListResult parseDiveLogsDeDiveList(const QByteArray &xmlData)
 		// process <DiveDates>
 		seenDiveDates = true;
 		while (reader.readNextStartElement()) {
-			if (reader.name() != "date") {
+			if (nameCmp(reader, "date") != 0) {
 				// qDebug() << "Skipping" << reader.name();
 				continue;
 			}
+#if QT_VERSION >= QT_VERSION_CHECK(5, 10, 0)
+			QStringView id = reader.attributes().value("divelogsId");
```

Comment:
Hint:
```c++
    auto id = reader.attributes().value("divelogsId");
```

Reply:
brilliant. I should have thought of that. 
YES.